devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -278,22 +258,6 @@ private List readDeviceBindingTable() {
 	["zdo mgmt-bind 0x${device.deviceNetworkId} 0", "delay 200"]
 }
 
-def shouldInvertLiftPercentage() {
-	return isIkeaKadrilj() || isIkeaFyrtur() || isSomfy()
```

Comment:
I don't think you can remove this yet because of the Somfy device.

Reply:
You are totally right, thanks for pointing that out.
Fixed.